proxy/package-lock.json

```diff
@@ -80,35 +80,34 @@
       }
     },
     "@babel/helper-member-expression-to-functions": {
-      "version": "7.13.0",
-      "resolved": "https://registry.npmjs.org/@babel/helper-member-expression-to-functions/-/helper-member-expression-to-functions-7.13.0.tgz",
-      "integrity": "sha512-yvRf8Ivk62JwisqV1rFRMxiSMDGnN6KH1/mDMmIrij4jztpQNRoHqqMG3U6apYbGRPJpgPalhva9Yd06HlUxJQ==",
+      "version": "7.13.12",
```

Comment:
What is the "proxy" in this context?

Reply:
The architecture of the upstream KUI code uses the term "proxy" for the sever side web app that in our case runs in the container.  (For the desktop KUI Electron application it runs it all together similar to the Slack Electron desktop application.) This results in two NodeJS node module dependency trees in our implementation.  One for the NodeJS web app (proxy) and one for the javascript UI code that is sent to and run in the browser.